packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/components/DrawingToolMarks/DrawingToolMarks.js

```diff
@@ -12,6 +12,15 @@ function DrawingToolMarks ({ activeMarkId, marks, onDelete, onDeselectMark, onFi
     const MarkingComponent = observer(mark.toolComponent)
     const ObservedDeleteButton = observer(DeleteButton)
     const isActive = mark.id === activeMarkId
+    const ref = React.createRef()
+    
+    function onFinishWithRef (event) {
+      onFinish(event, ref)
```

Comment:
It might be safer to pass `ref.current` here. Otherwise, you’re passing a reference to one of this component’s internal variables.

Reply:
👍 Good point, I've addressed that in the most recent commit.